docs/events/timeline-templates.asciidoc

```diff
@@ -1,5 +1,5 @@
 [[timeline-templates-ui]]
-= About Timeline templates
+= Use Timeline templates
```

Comment:
The verb is definitely more active than "About," but "Use" is still pretty generic and doesn't add a lot (most topics are about how to "use" some feature). Maybe keep it simple with no verb?
```suggestion
= Timeline templates
```

Reply:
I'm fine with either, really. Outside of the reasons that I listed in the PR description, I made this change to align this with the pattern used for the Cases and Osquery sections (top-level topic title is the feature name and the sub-topics are use-case-oriented).